applications/vessel-history/src/features/search/AdvancedSearch.tsx

```diff
@@ -60,6 +47,13 @@ const AdvancedSearch: React.FC = () => {
     [dispatchQueryParams]
   )
 
+  const onSearchClick = useCallback(
```

Comment:
why do we need to create a memoized function for something which doesn't have params?
I'd use `fetchResults` directly [here](https://github.com/GlobalFishingWatch/frontend/pull/973/files#diff-3f1966ff8a700843a0d4e57620497c7b9f85614237dbb24c3a1c0f74981747afR161) to simplify it

Reply:
Because if I use directly `fetchResults` then the  click `event` is passed as the first argument when `fetchResults` is expecting the `offset: number` and the app crashes and throws an error, that was my first take yesterday but it took me some time to find what was causing the problem 🤯  🤦